### PR TITLE
Add documentation for cron tasks endpoints

### PR DESCRIPTION
**Pull request by TheoEb**
Related to [CRON-48]

[CRON-48]: https://scalingo.atlassian.net/browse/CRON-48

### CODE REVIEW

File: _scalingo_api/cron_tasks.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+title: Cron tasks
```
Comment (EtienneM) [inline _scalingo_api/cron_tasks.md:2]: Title in English have first letters of all words in upper case
```suggestion
title: Cron Tasks
```

https://www.notion.so/scalingooriginal/Writing-Documentation-at-Scalingo-bb7985d6ea16455894417e36ede09cea

File: _scalingo_api/cron_tasks.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+layout: default
+---
+
+# Cron tasks
```
Comment (EtienneM) [inline _scalingo_api/cron_tasks.md:6]:
```suggestion
# Cron Tasks
```

File: _scalingo_api/cron_tasks.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+--- row ---
+
+**Cron tasks attributes**
+
+{:.table}
+| field                   | type    | description                                  |
+| ----------------------- | ------- | -------------------------------------------- |
+| command                 | string  | the cron command to be executed              |
```
Comment (EtienneM) [inline _scalingo_api/cron_tasks.md:15]: The `command` does not only contains the command, but also the cron expression

File: _scalingo_api/cron_tasks.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    },
+    {
+      "command": "0 13 * * * rake export:orders"
+    }
+  ]
+}
+```
+
+--- row ---
+## Get Cron tasks
```
Comment (EtienneM) [inline _scalingo_api/cron_tasks.md:38]:
```suggestion
## Get Cron Tasks
```

**Comment by Soulou** [comment]
Will be merged only when the feature is in prod

**Review by Soulou — COMMENTED** [review]
So @EtienneM by documenting this publicaly, it means that people will be able to update their Cron tasks autonomously, without considering the `cron.json` is that what we want?

**Comment by EtienneM** [comment]
I think we agreed not too make them publicly @TheoEb. Or did I missed something?

**Comment by TheoEb** [comment]
I think we want the Get to be public

**Comment by EtienneM** [comment]
Oh you're right, the GET can be public. It's used by the CLI anyway, so it is easy to detect this endpoint.

**Pull request merged**